dashboard-parent/dashboard-core/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,10 @@
+log4j.appender.Stdout=org.apache.log4j.ConsoleAppender
```

Comment:
`slf4j-simple` is already in charge
this file is not required

Reply:
Removed